user_tools/src/spark_rapids_pytools/cloud_api/emr.py

```diff
@@ -440,9 +440,9 @@ class EmrSavingsEstimator(SavingsEstimator):
     def _get_cost_per_cluster(self, cluster: EMRCluster):
         total_cost = 0.0
         for curr_group in cluster.instance_groups:
-            ec2_unit_cost = self.price_provider.catalog.get_value('ec2', curr_group.instance_type)
+            ec2_unit_cost = self.price_provider.catalogs['emr'].get_value('ec2', curr_group.instance_type)
```

Comment:
should we rename the catalog key to AWS? so the catalog is actually AWS that contains pricing components such as `EC2`, and `EMR`..etc

Reply:
Changes updated. Thank you!